backend/tracim_backend/tests/functional/test_comments.py

```diff
@@ -408,6 +408,16 @@ def test_api__post_content_comment__err_400__invalid_html(self, web_testapp) ->
         params = {"raw_content": "<p></p>Hello"}
         web_testapp.post_json("/api/workspaces/2/contents/7/comments", params=params, status=200)
 
+    def test_api__post_content_comment__err_400__user_not_member(
+        self, web_testapp, html_with_nasty_mention
+    ) -> None:
+        """
+        This test should raise an error as the html contains a mention to a user not member of the workspace
+        """
+        web_testapp.authorization = ("Basic", ("admin@admin.admin", "admin@admin.admin"))
+        params = {"raw_content": html_with_nasty_mention}
+        web_testapp.post_json("/api/workspaces/2/contents/7/comments", params=params, status=200)
```

Comment:
This test is wrong.

Reply:
Oh, this is actually an error that is present in the tests above!! I copy-pasted the previous one.